examples/mplot3d/trisurf3d_demo.py

```diff
@@ -27,6 +27,6 @@
 fig = plt.figure()
 ax = fig.gca(projection='3d')
 
-ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2)
+ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2, antialiased=True)
```

Comment:
antialiasing is turned off on pcolormesh by default to get around the polygon edge gap problem.

I'll just take the explicit `cmap` kwarg off here (making it effectively viridis).